datetime/bin/komand_datetime

```diff
@@ -6,8 +6,8 @@ from komand_datetime import connection, actions, triggers
 
 Name = 'Datetime'
 Vendor = 'rapid7'
-Version = '2.0.5'
-Description = 'This plugin manipulates timestamps using Python's maya library'
+Version = '2.0.6'
+Description = 'This plugin manipulates timestamps using Pythons maya library'
```

Comment:
Python's would be correct, but did you run into an issue building it with the apostrophe?

Reply:
I had problem, because "Python's" have apostrophe, same as in string: Description = 'This plugin manipulates timestamps using Pythons maya library'
So there is an error: SyntaxError: invalid syntax

We can change "Python's" to "Python`s"

Reply:
@mgorny-morecode We fixed this bug today in

```
$ icon-plugin --version
3.8.9
```

Run `tools/update-tools..sh` to get it.